css/main.css

```diff
@@ -133,4 +135,9 @@ ul {
     .slider-button{
         display: block;
     }
+}
+@media (min-width:1080px){
+    .container{
+        width: 89%;
```

Comment:
is it possible not to use percentage?

Reply:
I'm using percentage here for a more adaptive look of the 770px-1037px layout.